mne/report/tests/test_report.py

```diff
@@ -649,3 +624,98 @@ def test_survive_pickle(tmpdir):
     report.parse_folder(tempdir, render_bem=False)
     save_name = op.join(tempdir, 'report.html')
     report.save(fname=save_name, open_browser=False)
+
+
+@testing.requires_testing_data
+def test_full_report(tmpdir):
+    """Simulate user manually creating report by adding one file at a time."""
+    r = Report(title='My Report')
+    raw = read_raw_fif(raw_fname)
+    events = read_events(events_fname)
+    epochs = Epochs(raw=raw, events=events)
+    evokeds = read_evokeds(evoked_fname)
+
+    r.add_raw(raw=raw_fname, title='my raw data', tags=('raw',))
+    r.add_events(events=events_fname, title='my events',
+                 sfreq=raw.info['sfreq'])
+    r.add_epochs(epochs=epochs, title='my epochs', tags=('epochs',))
+    r.add_evokeds(evokeds=evokeds[:2], noise_cov=cov_fname,
+                  titles=['my evoked 1', 'my evoked 2'], tags=('evoked',))
+    r.add_ssp_projs(info=raw_fname, projs=ecg_proj_fname, title='my proj',
+                    tags=('ssp', 'ecg'))
+    r.add_ssp_projs(info=raw_fname, title='my proj', tags=('ssp'))
+    r.add_covariance(cov=cov_fname, info=raw_fname, title='my cov')
+    r.add_forward(forward=fwd_fname, title='my forward', subject='sample',
+                  subjects_dir=subjects_dir)
+
+    if get_3d_backend() is not None:
+        r.add_trans(trans=trans_fname, info=raw_fname, title='my coreg',
+                    subject='sample', subjects_dir=subjects_dir)
+        r.add_bem(subject='sample', subjects_dir=subjects_dir, title='my bem')
```

Comment:
This will probably take too long. We should make sure to heavily decimate the slicing here and elsewhere in new tests to minimize testing time.

Reply:
I'm decimating now much more (factor 100), so this shouldn't be much of a problem anymore, I hope